modules/truereachBidAdapter.js

```diff
@@ -0,0 +1,128 @@
+import * as utils from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { config } from '../src/config.js';
+import { BANNER } from '../src/mediaTypes.js';
+
+const SUPPORTED_AD_TYPES = [BANNER];
+const BIDDER_CODE = 'truereach';
+const BIDDER_URL = '//ads.momagic.com/exchange/openrtb25/';
```

Comment:
BIDDER_URL must use `https`

Reply:
@Fawke We have updated BIDDER_URL to use `https` always. Please verify.